Makefile

```diff
@@ -10,20 +10,70 @@ CHART_VERSION?=$(subst v,,$(GIT_TAG))
 KUBE_VERSION?="v1.22.7"
 CLUSTER_NAME?="operator-e2e"
 RAWCOMMITDATE=$(shell git log -n1 --format="%at")
-COMMITDATE?=$(shell date -d @"${RAWCOMMITDATE}" "+%FT%TZ")
+
+ifeq ($(shell go env GOOS),darwin) # Use the darwin/amd64 binary until an arm64 version is available
+	KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path --arch amd64 $(KUBEBUILDER_ENVTEST_KUBERNETES_VERSION))
+	COMMITDATE?=$(shell gdate -d @"${RAWCOMMITDATE}" "+%FT%TZ")
```

Comment:
gdate?

Reply:
it gnu date, `date` doesn't work on my mac with flags used in makefile. Does it work for you?

Reply:
Nope, but I also don't have gdate installed